lib/ansible/modules/database/postgresql/postgresql_privs.py

```diff
@@ -826,41 +872,44 @@ def build_absent(self):
 
 
 def main():
+    argument_spec = postgres_common_argument_spec()
```

Comment:
I think options that exist in https://github.com/ansible/ansible/blob/fbaa6c33caab85a0f71c39142d0ea7ed71e30515/lib/ansible/module_utils/postgres.py#L53-L60 should be removed from this modules argspec.

Reply:
Thank you for this hint, I removed
```
     port=dict(type='int', default=5432), 
     ssl_mode=dict(default='prefer', choices=['disable', 'allow', 'prefer', 'require', 'verify-ca', 'verify-full']), 
     ca_cert=dict(aliases=['ssl_rootcert'])
```

Reply:
this module seems very old, it contains several params with non-standard names.
I wouldn't like to touch them in this PR